build/docker/centos7/build/Dockerfile

```diff
@@ -200,6 +200,14 @@ RUN source /opt/rh/devtoolset-8/enable && \
     cd .. && \
     rm -rf /tmp/*
 
+# download old fdbserver binaries
```

Comment:
Why does this need to go to the `build` image, not `devel` image?
Can we exclude this section when the platform is not Linux x86_64?

Reply:
these files were not architecture specific before this change, this is not the time to add that feature.